Cycling Analysis/Strava importer.ipynb

```diff
@@ -1270,28 +1359,48 @@
     "\n",
     "\n",
     "# Process each FIT file\n",
-    "#for index, row in df.iterrows():\n",
-    "for i, item in enumerate(df.iterrows()):\n",
-    "    row = item[1]\n",
-    "\n",
-    "    print (f'Processing {row[\"Filename\"]} ({i+1} of {len(df)}).')\n",
+    "for i, (index, row) in enumerate(df.iterrows()):\n",
+    "    print(f'Processing {row[\"Filename\"]} ({i + 1} of {len(df)}).')\n",
     "    fit_file_path = os.path.join(source_path, row['Filename'])\n",
     "    csv_file_path = os.path.join(data_directory, row['Filename']).replace('.fit', '.csv')\n",
     "\n",
-    "    fit_gz_to_csv_gz(fit_file_path, csv_file_path)\n",
```

Comment:
We can implement try expect blocks to handle exceptions that may occur during file processing.

Reply:
Hi again, this is part of a Notebook so any errors are presented to the person running the code to fix. This is expected to be run interactively so try/catch clauses would only repeat the error to the user.

This code will not be available to end users - it will only be run by backend developers.